### DISCUSSION TITLE
[5~6주차 회의록]안건 결정

### DISCUSSION THREAD

**Discussion by Chominsu98** [PocketBand/frontend-web#18] [회의록]
## 우리 서비스의 핵심 특징
- 밴드매니저(자체관리)+ 합주추억 공간 / 서브(커뮤니티)
- 2차 오픈시에는 진짜 우리만의 기능들을 넣어보는걸 고려해보자(ex) 영상 자동 녹화 중단, 오디오 악기별 분리 가능 기능 등 

## 고려 필요 사항
- (AS-IS)밴드 추가/탈퇴 화면을 설정화면에서 지정 중-->(To-Be)프로필쪽으로 2차로 이동하는거 어떨지..?
-  (AS-IS) 밴드 추가라는 명을 사용 중 -->(To-Be)밴드 가입/개설로 바꾸는게 어떨지?

## 기술적 연구 사항
1) 영상 압축 인코딩 방식
2) 업로드 속도
3) 랜더링 속도

- 1차 오픈시에는 영상이 안되면 사진으로 우선 오픈해보는건 또 어떨지 고려해보자

## 전체 회의 안건
- CMN002 검색했을때 정렬 기준
>합주실) 정렬기준: (평점 ,리뷰 등 )복합요소로 지표 점수로 정렬
>게시물) 정렬기준: (조회수,좋아요 수,즐겨찾기 수) 백엔드 자체 결정

- CMN003 알림 유지 일수(알림창 db 관련)
>1차적으로는 그냥 유지

- 회원가입 시 아이디 중복 체크
> (닉네임은 중복 금지) 

- CMN006-2 회원가입 시 주소 검색 api
> 공공 도로명주소를 사용하여 (동)(읍)(면)을 서치해서 드롭다운으로 해서 선택

- COM001 커뮤니티-투데이레슨에 추천되는 콘텐츠를 회원가입 시에 작성한 악기 정보 위주로? 할것인지 검토
> 컨텐츠 : 경력 + 악기 (유튜브 api 검색으로 넘김)
각 경력마다 수준을 분류하여 검색결과를 같이 조합하여 넘김
>악기별로 골라볼수도 있다
> ex) 피아노 경력 1년이하 : 초보라는 단어를 매핑해서 같이 유튜브에 넘겨줌 이런식

- COM003 / MAP002 리뷰 전체 개수 - 자체리뷰만? or 전체(카카오+네이버+자체리뷰)
> 카카오 리뷰 수+ 네이버 리뷰 수 + 자체 리뷰 수
> 카카오 평점+ 네이버 평점 + 자체 평점
> 법적인 요소는 우선 나중에 고려해보고 도입을 해보자

- MAP001 현위치기준 정렬 평점순 - 자체평점만? or 전체(카카오+네이버+자체평점)
> 카카오 리뷰 수+ 네이버 리뷰 수 + 자체 리뷰 수
> 카카오 평점+ 네이버 평점 + 자체 평점

- PLU002 영상 업로드
> 백엔드 협의 필요

- 밴드 탈퇴 or 강퇴 시 사용자프로필 밴드탭에서 예전 밴드 피드를 없앨건지?
> 사용자 프로필 상에서 밴드쪽으로 올라간 영상은 개인이 개인탭쪽으로 이동해서 조회될 수 있게 하며
밴드탭에서 업로드시 개인공유 권한부여자를 선택한 후 탈퇴시 개인은 밴드탭에서
 (공유버튼을 추가해줘야함)
(사용자프로필)밴드탭쪽에서는 탈퇴 밴드이므로 안보임
밴드 페이지에서는 탈퇴 사용자가 포함된 영상은 그대로 남는다

## 설계서 수정 필요 사항
- 영상 업로드 시 권한 부여자에 대한 선택이 가능하도록 추가 옵션이 필요함 **(정상)PLU002-1**
- 영상 업로드시 태그랑 합주,개인 버튼의 중복으로 하나만 있게 수정 **(정상) PLU002-1**
- 게시물 상세 쪽에서 상단 신고하기(2차내용)버튼에 권한 부여자에 한하여 개인 피드로 공유하기 버튼 추가  **(민수)COM002-1**

## [2차 개발로 연기]
- 컨텐츠 신고 기능, 추천에서 제외시키기
- COM003-1 /MAP002-1  합주실 정보 수정 신고

**Comment by Oneul1213** [comment]
PLU002-1 수정 완료
